l1-contracts/contracts/state-transition/ValidatorTimelock.sol

```diff
@@ -3,8 +3,8 @@
 pragma solidity 0.8.20;
 
 import "@openzeppelin/contracts/access/Ownable2Step.sol";
+import "./chain-interfaces/IExecutor.sol";
 import "./libraries/LibMap.sol";
-import "./interfaces/IExecutor.sol";
 
 /// @author Matter Labs
 /// @custom:security-contact security@matterlabs.dev
```

Comment:
NIT: comment need to be updated

Reply:
What should I update here?

Reply:
All mentions of zkSync should be changed in favor of hyperchains

Reply:
I'm not sure I got all, but got quite a few